### PR TITLE
Updated documentation for current interactors

### PR DESCRIPTION
**Pull request by bsekachev**
### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@TOsmanov 
Could you please review the PR? Can we improve it somehow?

File: site/content/en/docs/manual/advanced/ai-tools.md [hunk 1]
```diff
@@ -44,7 +45,33 @@ In most cases specifying positive points alone is enough to build a polygon.
...
+This is an optimized version of the original model, introduced at the end of 2017.
+It uses the information about extreme points of an object to get its mask. The mask then converted to a polygon.
+For now this is the fastest interactor on CPU.
+
+![](/images/dextr_example.gif)
+
+### Feature backpropagating refinement scheme (f-BRS)
+
+The model allows to get a mask for an object using positive points
+(should be clicked on the foreground), and negative points
```
Comment (TOsmanov) [inline site/content/en/docs/manual/advanced/ai-tools.md:59]:
```suggestion
(should be left-clicked on the foreground), and negative points
```

File: site/content/en/docs/manual/advanced/ai-tools.md [hunk 2]
```diff
@@ -44,7 +45,33 @@ In most cases specifying positive points alone is enough to build a polygon.
...
+(should be clicked on the background, if necessary).
```
Comment (TOsmanov) [inline site/content/en/docs/manual/advanced/ai-tools.md:60]:
```suggestion
(should be right-clicked on the background, if necessary).
```

File: site/content/en/docs/manual/advanced/ai-tools.md [hunk 3]
```diff
@@ -44,7 +45,33 @@ In most cases specifying positive points alone is enough to build a polygon.
...
+It is recommended to run the model on GPU, if possible.
+
+![](/images/fbrs_example.gif)
```
Comment (TOsmanov) [inline site/content/en/docs/manual/advanced/ai-tools.md:63]: I think you can cut out the footage of waiting in this gif

Comment (bsekachev) [reply site/content/en/docs/manual/advanced/ai-tools.md:63]: Done

**Pull request merged**